app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/ProfileFragmentTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.github.sdp.tarjetakuna.ui
+
+import android.content.Context
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.closeSoftKeyboard
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.sdp.tarjetakuna.R
+import com.github.sdp.tarjetakuna.ui.profile.ProfileFragment
+import com.github.sdp.tarjetakuna.utils.SharedPreferencesKeys
+import com.github.sdp.tarjetakuna.utils.SharedPreferencesKeys.shared_pref_name
+import junit.framework.TestCase.assertEquals
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ProfileFragmentTest {
```

Comment:
These tests are nice but they're missing "negative" cases, such as what happens if the user tries to input an empty name